docs/content/en/docs/Reference/image_matrix.md

```diff
@@ -23,6 +23,20 @@ The installable mediums part of the releases are generated with the methods desc
 | **openSUSE based (RaspberryPi 3 and 4, arm64)**     	| https://quay.io/repository/kairos/core-opensuse-arm-rpi 	| https://quay.io/repository/kairos/kairos-opensuse-arm-rpi 	|
 | **Alpine Linux based (RaspberryPi 3 and 4, arm64)** 	| https://quay.io/repository/kairos/core-alpine-arm-rpi   	| https://quay.io/repository/kairos/kairos-alpine-arm-rpi   	|
 
-<Info>
+{{% alert title="Note" color="info" %}}
 ** the `ubuntu` flavor tracks the latest available Ubuntu release (at the time of writing 22.10). the LTS flavors instead are tracking the latest LTS available in dockerhub. i.e. ubuntu-22-lts uses 22.04 as base image
-</Info>
\ No newline at end of file
+{{% /alert %}}
+
+{{% alert title="Note" color="info" %}}
+  The pipelines don't publish `img` artifacts for arm architecture because the files are too big for Github Actions (they are above the artifact size limit).
```

Comment:
should we reference to that also on the raspberry page? The current docs guide to downloading assets https://kairos.io/docs/installation/raspberry/